mods/weather/init.lua

```diff
@@ -6,6 +6,10 @@ if mg_name == "v6" or mg_name == "singlenode" or
 	return
 end
 
+-- Public behavior flags
```

Comment:
there's an if right above here that disables the mod for v6 or singlenode ... which would result in no shadows
not intended I bet